mava/components/jax/building/parameter_client.py

```diff
@@ -238,7 +238,7 @@ def name() -> str:
 ###################
 
 
-class ExecutorParameterClientSeparateNetworks(BaseParameterClient):
+class ExecutorParameterClientSeparateNetworks(ExecutorParameterClient):
```

Comment:
I get why you did this but the code in this class is exactly the same as ExecutorParameterClient besides when we create the networks so if we extend this we should reuse code or extend base

Reply:
Nice catch @AsadJeewa :smile: I updated the code. Does it look better now?

Reply:
I am happy